study.md

```diff
@@ -22,7 +22,7 @@ In your own words, explain one reason why we use "this". In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+Use this to refer to an object that doesn't exist until runtime.
```

Comment:
You're close here. When we call a method on an object, `this` will refer to that object. It isn't that the object doesn't exist, its just that we won't know what the object is until runtime.